packages/survey-creator-core/src/creator-base.ts

```diff
@@ -119,56 +119,59 @@ export class CreatorBase extends Base
    * Specifies whether to display the Designer tab.
    *
    * Default value: `true`
+   * @see activeTab
+   * @see onDesignerSurveyCreated
    */
   @property({ defaultValue: true }) showDesignerTab: boolean;
   /**
    * Specifies whether to display the JSON Editor tab.
    *
    * Default value: `true`
+   * @see activeTab
    */
   @property({ defaultValue: true }) showJSONEditorTab: boolean;
-  /**
-   * Obsolete. Please use showPreviewTab property
-   * @see showPreviewTab
-   */
+
   @property({ defaultValue: true }) showTestSurveyTab: boolean;
   /**
    * Specifies whether to display the Preview tab.
    *
    * Default value: `true`
+   * @see activeTab
+   * @see onPreviewSurveyCreated
    */
   public get showPreviewTab(): boolean { return this.showTestSurveyTab; }
   public set showPreviewTab(val: boolean) { this.showTestSurveyTab = val; }
   /**
    * Specifies whether to display the Themes tab.
    *
    * Default value: `false`
+   * @see activeTab
+   * @see saveThemeFunc
    */
   @property({ defaultValue: false }) showThemeTab: boolean;
   /**
    * Specifies whether to display the Translation tab.
    *
    * Default value: `false`
+   * @see activeTab
    */
   @property({ defaultValue: false }) showTranslationTab: boolean;
   /**
    * Specifies whether to display the Logic tab.
    *
    * Default value: `false`
+   * @see activeTab
    */
   @property({ defaultValue: false }) showLogicTab: boolean;
   @property({ defaultValue: false }) useTableViewInLogicTab: boolean;
-  /**
-   * Set delay for page hover
-   */
   @property({ defaultValue: 200 }) pageHoverDelay: number;
   /**
-   * You need to set this property to true if you want to inplace edit item values instead of texts.
+   * Allows users to edit choice values instead of choice texts on the design surface.
```

Comment:
Hides the text column for choices, rateValues properties, columns and rows properties for single answer matrix question and rows properties for multiple answer matrix question